tools/tune/eq/mls_freq_resp.m

```diff
@@ -57,8 +57,8 @@
 else
 	selftest = 0;
 end
-measfn = sprintf('mls-%s.wav', id);
-csvfn = sprintf('mls-%s.txt', id);
+measfn = sprintf('mls-%d.wav', id);
+csvfn = sprintf('mls-%d.txt', id);
```

Comment:
id is supposed to be text, why number?

Reply:
I preferred a name like id = 'notebook_x' to have more meaningful file names. Especially the .csv that is a report is useful to be an easy name to remember.

Reply:
fine, I will provide a name . Thanks